### DISCUSSION TITLE
Add google gemma model

### DISCUSSION THREAD

**Discussion by lwpk110** [lwpk111/llm-learning#1] [Q&A] [answered]
Overview
Gemma is a family of lightweight, state-of-the art open models built from the same research and technology used to create the Gemini models.

Large Language Models (LLMs) like Gemma have been shown to be effective at a variety of NLP tasks. An LLM is first pre-trained on a large corpus of text in a self-supervised fashion. Pre-training helps LLMs learn general-purpose knowledge, such as statistical relationships between words. An LLM can then be fine-tuned with domain-specific data to perform downstream tasks (such as sentiment analysis).

LLMs are extremely large in size (parameters in the order of millions). Full fine-tuning (which updates all the parameters in the model) is not required for most applications because typical fine-tuning datasets are relatively much smaller than the pre-training datasets.

**Comment by lwpk111** [comment]
ok，

**Comment by lwpk110** [comment, marked as answer]
the page of tutorial is `https://www.kaggle.com/models/google/gemma/frameworks/keras/variations/gemma_instruct_7b_en`
